guides/source/api_documentation_guidelines.md

```diff
@@ -101,6 +101,12 @@ Oxford Comma
 Please use the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma)
 ("red, white, and blue", instead of "red, white and blue").
 
+Wrap at 80 Characters
```

Comment:
I don't think we did these. In the past we also allowed long running lines if it made more sense to group together contextually.

The 80 character limit also seems a bit outdated now(Used to be based on horizontal rule size on GitHub). Maybe can be 120? 

We can mention though that this is for one new documentation and not existing? We'd get too many new cosmetic PRs

Reply:
I'm fine with 100 or 120 as well, but I've seen 80 mentioned in some reviews.
But having a number as a guide makes it easier for reviewers.

I've added the mentioning of "new documentation".

Reply:
I'd make another vote for 120.

Reply:
I am ok with 80 for text documentation; I feel it is better for reading, and It allows work with a split window, even on small screens. 

I think that it makes sense to add some examples or directives of how to handle long code lines on different scenarios:

- URLs within a code example; I would keep them as long as necessary.
- Bash lines (rake, for example) it's easy to wrap with `\`.
- Code examples, it's a bit hard to keep the line length restriction and a coherent indentation, at least with my vim setup.

In any case, I am fine with whatever the core team decides. It's good to have a reference.